study_guides/react_redux/react_redux_cycle_steps.md

```diff
@@ -23,4 +24,6 @@ ui:{
     * the route will also pass props to the components they render.
 5. after invoking msp and mdp we will then render the component.
     * if there is a componentDidMount() then we will hit that after the component has attempted to render. 
-6. if there is a prop that dispatches an ajax request in the componentDidMount() we invoke a function that dispatches the ajax request to the middleware. the middle ware will then see if the the action is a function or not.  If it is a function it will then invoke the function and then pass the dispatch function as an argument. the dispatch function that is passed will then be used after we get the info back from the ajax request. remember that the dispatch sends an action to the middleware and then invokes the next argument that ships it to the next middleware.  if there is no other middle ware then we will send the action to all of the reducers. 
+
+# middleware summary
+if there is a prop that dispatches an ajax request in the componentDidMount() we invoke a function that dispatches the ajax request to the middleware. the middle ware will then see if the the action is a function or not.  If it is a function it will then invoke the function and then pass the dispatch function as an argument. the dispatch function that is passed will then be used after we get the info back from the ajax request. remember that the dispatch sends an action to the middleware and then invokes the next argument which then ships it to the next middleware.  if there is no other middleware then we will send the action to all of the reducers. 
```

Comment:
suggestion for better readability/to be more clear: 

If there is a prop that dispatches an ajax request in the componentDidMount(), we are, essentially, invoking a function that dispatches this ajax request to the middleware. The middleware will then see if the the action is a function or not.  If the action is a function, the middleware will invoke that function and then pass the dispatch function as an argument. The dispatch function that was passed along will then be used after we get our data response from the ajax request. Remember that the dispatch sends an action to the middleware and then invokes the next argument which then ships the action to the next middleware.  If there is no other middleware then we will send the action to all of the reducers, through the root reducer.

Reply:
Thanks for the suggestion! i just reworded with what you put.  Can you please take another look just in case i missed anything. @mkayys